xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -463,7 +465,8 @@
       "properties": {
         "name": { "type": "string", "required": true },
         "role": { "type": "string", "required": true },
-        "artistid": { "type": "integer", "required": true }
+        "roleid": { "$ref": "Library.Id", "required": true },
```

Comment:
Can you rename the type from `Audio.Contributors` to `Audio.Roles` to fit the name of the `GetRoles` method?